app/src/main/res/menu/drawer_menu.xml

```diff
@@ -4,21 +4,27 @@
     <item
         android:id="@+id/home"
         android:icon="@drawable/ic_home"
-        android:title="@string/home_title"/>
+        android:title=""/>
```

Comment:
We use 
```
navigationView.setTextVisibility(false)
```
So, we can save titles here in `drawer_menu.xml`. It can be helpful for accessibility

Reply:
There was a bug on API 15 that displays titles over icons.